packages/panels/resources/views/components/layout/base.blade.php

```diff
@@ -78,6 +78,16 @@
             </script>
         @endif
 
+        @if (! filament()->hasDarkMode())
+            <script>
+                const theme = localStorage.getItem('theme') ?? 'system'
+                
+                if (theme != 'light') {
+                    localStorage.setItem('theme', 'light')
+                }
+            </script>
```

Comment:
We just need `localStorage.setItem('theme', 'light')`, right?

Reply:
That's correct, too